packages/gated-content/package.json

```diff
@@ -45,6 +45,9 @@
   },
   "license": "MIT",
   "dependencies": {
+    "@ethersproject/abi": "^5.7.0",
+    "@ethersproject/address": "^5.7.0",
+    "@ethersproject/bignumber": "^5.7.0",
```

Comment:
Unless something is changed these should stay as peer dependency only. Maybe there is an issue somewhere else, can you please clarify what made you make this change?